hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ScanOptions.java

```diff
@@ -64,4 +66,13 @@ default void readAllVersions() {
   void setKeepDeletedCells(KeepDeletedCells keepDeletedCells);
 
   KeepDeletedCells getKeepDeletedCells();
+
+  int getMinVersions();
+
+  void setMinVersions(int minVersions);
+
+  /**
+   * Returns a read-only copy of the Scan object
```

Comment:
The comment is a bit confusing as you are free to call the modification methods of the returned Scan object? So it is not 'read only'.

Maybe better with "The returned Scan object is just a copy, modifying it will have no effect, so treat it as read-only."

Of course I'm not a native English speaker, so feel free to use your own word. Just show the meaning that 'you can modify, but no effect, so please do not modify'.

Reply:
Good point. Done.